src/org/omegat/core/machinetranslators/ApertiumTranslate.java

```diff
@@ -93,8 +105,17 @@ protected String translate(Language sLang, Language tLang, String text) throws E
         String sourceLang = apertiumCode(sLang);
         String targetLang = apertiumCode(tLang);
 
-        String url2 = GT_URL2.replace("#sourceLang#", sourceLang).replace("#targetLang#", targetLang);
-        String url = GT_URL + URLEncoder.encode(trText, "UTF-8") + url2;
+        String server = getCredential(PROPERTY_APERTIUM_SERVER_URL);
+        String apiKey = getCredential(PROPERTY_APERTIUM_SERVER_KEY);
+
+        if (!useCustomServer()) {
+            server = APERTIUM_SERVER_URL;
+        }
+        if (!useCustomServer()) {
+            apiKey = APERTIUM_SERVER_KEY;
+        }
```

Comment:
These two if statements can be combined.

Reply:
Combined in latest commit, no reason to keep them separated.